Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
I'm guessing something in your GOPATH is owned by root. What happens if you `sudo chown -R $(whoami) $GOPATH` and try again?